ui/app/components/clients/dashboard.js

```diff
@@ -60,36 +60,32 @@ export default class Dashboard extends Component {
     return !isSameMonth(new Date(this.startTime), new Date(this.endTime));
   }
 
-  // Determine if we have client count data based on the current tab
-  get hasClientData() {
-    if (this.args.tab === 'current') {
-      // Show the current numbers as long as config is on
-      return this.args.model.config?.enabled !== 'Off';
-    }
-    return this.args.model.activity && this.args.model.activity.total;
+  // top level TOTAL client counts from response for given date range
```

Comment:
api returns a total object for the date range queried, this is that object
`total: {
clients: 500,
entity:clients: 300,
non_entity_clients: 200}`